ArjunaCore/arjuna/tests/classes/com/hp/mwtests/ts/arjuna/atomicaction/AtomicActionAsyncTest.java

```diff
@@ -49,6 +56,28 @@ public void testAbort() throws Exception {
     }
 
     @Test
+    public void testAbortAsyncAC() throws Exception {
```

Comment:
I have updated the test to reflect the issue being raised (i.e. rollback causes NPE)